tests/commit.bats

```diff
@@ -96,3 +96,26 @@ load helpers
   cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
   run_buildah commit --signature-policy ${TESTSDIR}/policy.json $cid
 }
+
+@test "commit with name" {
+  buildah from --name nginxc nginx
```

Comment:
Why so many different source images? Each time you specify `buildah from X/Y/Z` you need to pull X/Y/Z. Let's please be consistent and stick with common, small images.